entity-types/ext-cacti/definition.yml

```diff
@@ -0,0 +1,20 @@
+domain: EXT
+type: CACTI
+synthesis:
+  rules:
+  - identifier: hostname
+    name: hostname
+    encodeIdentifierInGUID: true
+    conditions:
+    - attribute: eventType
+      value: PingLatencySample
+    - attribute: metricName
+      value: "value"
+    tags:
+      agentVersion:
```

Comment:
For consistency with other entities, this tag should be `newrelic.agentVersion`: https://github.com/search?q=repo%3Anewrelic%2Fentity-definitions%20newrelic.agentVersion&type=code

Reply:
For cacti using nri-flex, I am not receiving the `newrelic.agentVersion` value. I am getting it with the `agentVersion` name. Is this sufficient, or should I replace it with a different parameter?

Reply:
In the link I shared there are examples of tag renaming. Find here another example of `agentVersion` being renamed to tag name `newrelic.agentVersion`: https://github.com/newrelic/entity-definitions/blob/9be1965d0139b036bb0bee32a1164d0592459a0e/entity-types/infra-container/definition.yml#L230-L231